docs/config.md

```diff
@@ -265,25 +319,33 @@ Configures how many AutoNAT requests to service per `AutoNAT.Throttle.Interval`.
 
 Default: 30
 
+Type: `integer`
```

Comment:
Most of the fields with type `integer` do not have any specifications on what the valid ranges of the field are and what, if any, special values exist. It'd be fantastic to have those added in, although given that it's unrelated to the transport configs there's no reason to block on it.

For example, I'm pretty sure <=0 here means the limit is disabled (i.e. is equivalent to infinity) with 0 being used by default in the code to indicate disabled.

It's worth noting that many of these integers have the same (or nearly the same) semantics as `priority`, and it might be worth renaming `priority` to something more broad (I don't have any good name ideas at the moment, so feel free to ignore unless you have a better name).

Reply:
I've updated the documentation.